workshop/Lab3/README.md

```diff
@@ -0,0 +1,252 @@
+# Lab 3: Scale and update apps natively, building multi-tier applications.
+
+In this lab, understand how to build applications using basic building blocks which are Kubernetes objects such as pods, deployments and services. These objects are defined in configuration files using yaml. Once defined, these objects are created using Kubernetes CLI. In this lab we will explore how to build a multi-tier application that can scale.
```

Comment:
```
In this lab you'll learn how to deploy the same guestbook application we deployed in the previous labs, however, instead of using the `kubectl` command line helper functions we'll be deploying the application using configuration files. The configuration file mechanism allows you to have more fine-grained control over all of resources being created within the Kubernetes cluster.
```

Reply:
yea, and I'm wondering if we should change it since 80 will feel more 'normal' and 'real'